docs/http/3.http-basic/URI.md

```diff
@@ -0,0 +1,56 @@
+---
+title: URI的组成部分
+sidebar_label: URI的组成部分
+---
+
+## URI的定义
+
+URI（Uniform Resource Identifier，统一资源**标识符**）是一个用于**标识某一互联网资源名称**的**字符串**。
+
+## URI和URL的关系
+
+与URI相比，显然我们更熟悉的是URL（Uniform Resource Locator，统一资源**定位符**），例如我们的浏览器中的网址就是URL。那么这两者之间有什么关系呢？如何理解`标识符`和`定位符`这两者的含义呢？
+
+我们可以明确的一点是**URL是URI的子集，我们能看到的URL都是URI**，而URL只是URI的一种表示方式。
+
+举个外卖员送外卖的例子，外卖员可以通过拨打手机号通知你外卖送到了，门卫不让进，到小区门口取一下。外卖员也可以通过外卖订单上的地址然后敲门将卖外送给你。此时你就是外卖员要找的“互联网资源”，你的手机号就是你的“标识符”，而你的地址就是你的“定位符”。所以不论是用地址**定位**的方式还是用手机号编号的方式，我们都可以唯一确定一个人，都是URI的一种实现，**而URL就是用地址定位的方式实现的URI**。
+
+- **URI的例子**
+```
+http://example.org/absolute/URI/with/absolute/path/to/resource.txt
+urn:issn<XSLT>:1535-3613</nowiki>
```

Comment:
You'd better add more details about `urn` if use this example or remove `urn:issn`